src/pages/home-page/home-page.js

```diff
@@ -24,11 +28,9 @@ const HomePage = () => {
   const commonStyles = useCommonStyles();
 
   const dispatch = useDispatch();
+  const classes = useStyles();
+  const { loading, photos } = useSelector(HomePageSelector);
```

Comment:
can you rename to classes to styles, just to have the same variable name in all files?

Reply:
fixed